itou/www/siae_evaluations_views/tests/tests_siaes_views.py

```diff
@@ -225,6 +279,57 @@ def test_post_buttons_state(self):
         response = self.client.get(select_criteria)
         assert response.status_code == 403
 
+    def test_post_buttons_state_with_submission_freezed(self):
+        fake_now = timezone.now()
+        evaluated_job_application = create_evaluated_siae_with_consistent_datas(self.siae, self.user)
+
+        submit_disabled = """
+            <button class="btn btn-outline-primary disabled float-right">
+                Soumettre à validation
+            </button>
+        """
+        submit_active = """
+            <button class="btn btn-primary float-right">
+                Soumettre à validation
+            </button>
+        """
+        select_criteria = reverse(
+            "siae_evaluations_views:siae_select_criteria",
+            kwargs={"evaluated_job_application_pk": evaluated_job_application.pk},
+        )
+
+        self.client.force_login(self.user)
+
+        # criterion selected with uploaded proof
+        evaluated_administrative_criteria = EvaluatedAdministrativeCriteriaFactory(
+            evaluated_job_application=evaluated_job_application, proof_url="https://server.com/rocky-balboa.pdf"
```

Comment:
C’est la valeur par défaut. Comme tu préfères.
```suggestion
            evaluated_job_application=evaluated_job_application,
```

Reply:
Je préfère que ce soit clair qu'il y a un proof_url